xija/tests/test_get_model_spec.py

```diff
@@ -0,0 +1,81 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import re
+
+import pytest
+import requests
+import git
+
+from ..get_model_spec import (get_xija_model_spec, get_xija_model_names,
+                              get_repo_version, get_github_version)
+
+try:
+    # Fast request to see if GitHub is available
+    req = requests.get('https://raw.githubusercontent.com/sot/chandra_models/master/README',
+                       timeout=5)
+    HAS_GITHUB = req.status_code == 200
+except Exception:
+    HAS_GITHUB = False
+
+
+def test_get_model_spec_aca_3_30():
+    # Version 3.30
+    spec, version = get_xija_model_spec('aca', version='3.30')
+    assert spec['name'] == 'aacccdpt'
+    assert 'comps' in spec
+    assert spec["datestop"] == "2018:305:11:52:30.816"
+
+
+def test_get_model_spec_aca_latest():
+    # Latest version
+    spec, version = get_xija_model_spec('aca', check_version=HAS_GITHUB)
```

Comment:
I'm a little confused here.  I understand the point of checking the capability to check the version, but it seems like the test should pass whether the user has SKA/chandra_models up-to-date (or testing or whatever), as that's not something the module controls.  Seems like the test should make a temporary area for the chandra_models clone and run against that or the like.  Or am I misunderstanding?  Not a big deal either way.

Reply:
Yes, being pedantic you are right. In practice people should keep their local archive up to date and if they don't then this will fail, but we don't care.  It should always pass on a production / well-maintained machine.